index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+
+        <header class="header-container">
+            <h3 class="white">My Portfolio</h3>
+            <img src="./assets/icons/menu.png" alt="">
+        </header>
+        <section class="headline-container">
+            <h1 class="white"><em class="headline-title">Hey There I’m Aubin</em><br> I’am a Software Developer</h1>
+            <p class="white">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="social-media-icones-container">
+                <img src="/assets/icons/1.png" alt="">
```

Comment:
- [Optional] Consider making the list of social icons an unordered list. This is best for HTML semantics.
- Also wrap the icons in an `a` tag that links to the various social media handle. You can use:
```html
<ul>
  <li>
    <a href=""><img ></a>
  </li>
  <li>
    <a href=""><img ></a>
  </li>
</ul>
```